ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellFileAccessoryViewDemoController.swift

```diff
@@ -519,9 +519,8 @@ class TableViewCellFileAccessoryViewDemoController: DemoTableViewController {
 
```

Comment:
For sure outside the scope of this review, but this `extraPadding` line is weird. Doesn't seem like we should really have other, different constants in our demos? Unless they're explicitly token overrides.

Reply:
Yeah I don't know what's going on here 👎 but agreed, that's a "not this PR, but worth re-evaluating" problem.